website/addons/s3/static/s3NodeConfig.js

```diff
@@ -191,6 +191,13 @@ ViewModel.prototype.importAuth = function() {
 ViewModel.prototype.createCredentials = function() {
     var self = this;
     self.creatingCredentials(true);
+
+    if ( !(self.secretKey() && self.accessKey()) ){
+        var message = 'All those fields above are required.';
```

Comment:
Rather than imperatively checking the truthiness of these values, can we just use the `required` HTML attribute on the inputs?